src/components/Timeline/TextEditor/Toolbar.tsx

```diff
@@ -0,0 +1,46 @@
+import { makeStyles } from '@material-ui/styles';
+import React from 'react';
+import {
+  FormatBold,
+  FormatItalic,
+  FormatListBulleted,
+  FormatListNumbered,
+  FormatQuote,
+  FormatStrikethrough,
+} from '@material-ui/icons';
+
+import {
+  AddAttachmentButton,
+  AddLinkButton,
+  BlockButton,
+  MarkButton,
+  RemoveLinkButton,
+} from './ToolbarButtons';
```

Comment:
The `PascalCase` casing of this module (`ToolbarButtons`) implies to me based on the conventions used in the codebase in general that the module should export a default module called `ToolbarButtons`, which of course it doesn't. 

I think the consistent way to name it would be `toolbarButtons`.